operator/cmd/configurator/configurator.go

```diff
@@ -57,6 +57,8 @@ const (
 	redpandaRPCPortEnvVar                                = "REDPANDA_RPC_PORT"
 	svcFQDNEnvVar                                        = "SERVICE_FQDN"
 	additionalListenersEnvVar                            = "ADDITIONAL_LISTENERS"
+	bootstrapDestinationEnvVar                           = "BOOTSTRAP_DESTINATION"
```

Comment:
just my two cents but I think pulling these from CLI args would be a bit nicer for debuggers. We can punt this to a refactor later as the configurator desperately needs something to help read configs from CLI and/or Envvars

Reply:
Agreed. I just followed the existing pattern here which I guess is strongly "12-factor"-influenced, but I'd prefer something better here.